<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_Window</class>
 <widget class="QMainWindow" name="Login_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Edit_ID">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_Password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::InputMethodHint::ImhHiddenText|Qt::InputMethodHint::ImhNoAutoUppercase|Qt::InputMethodHint::ImhNoPredictiveText|Qt::InputMethodHint::ImhSensitiveData</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Text_ID">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>이 름 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text_Password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>학 번 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_Btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 144, 255);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>로그인 →</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>--------------------------------Or----------------------------------------------------------</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sinup_Btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 176, 176);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>회원가입</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
